doc/nrf/gs_installing.rst

```diff
@@ -44,27 +55,27 @@ The installation process is different depending on your operating system.
          :docset: zephyr
          :dedent: 6
          :start-after: .. _install_dependencies_windows:
-         :end-before: _get_the_code:
+         :end-before: #. Close the window and open a new
 
    .. group-tab:: Linux
 
-      To install the required tools on Ubuntu, complete the following steps:
+      To install the required tools on the Linux operating systems, see the :ref:`zephyr:linux_requirements` section in the Zephyr documentation.
```

Comment:
I don't like putting this sentence first. It leads back to the issue we had before that users will click the link and will not know when to return to this page.
That's why I put the Ubuntu instructions first and made it clear that you need the link ONLY for other distributions.
See NCSDK-7820.